crates/graphql/src/schema/scalars/public_key.rs

```diff
@@ -0,0 +1,71 @@
+use super::prelude::*;
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Default)]
+pub struct PublicKey(String);
```

Comment:
I wanna huddle on fixing this definition up, we can make it more type-safe without needing a macro to repeat definitions.